apps/full-site-editing/full-site-editing-plugin/donations/index.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable wpcalypso/import-docblock */
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { getBlockType, registerBlockType, unregisterBlockType } from '@wordpress/blocks';
+import { __, _x } from '@wordpress/i18n';
+/* eslint-enable wpcalypso/import-docblock */
+
+/**
+ * Appends a "paid" tag to the Donations block title if site requires an upgrade.
+ */
+const addPaidBlockFlags = async () => {
```

Comment:
I wonder if we could move this to a shared utils file so it can be reused here and in the premium content block.

Reply:
I thought that too, but wanted to see things working and in place first – could we leave that for a separate PR after this is merged?

Reply:
Sure 👍